sherpa/ui/utils.py

```diff
@@ -16858,13 +16796,8 @@ def image_xpaget(self, arg):
 
         Notes
         -----
-        The XPA access point [1]_ of the ds9 image viewer lets
-        commands and queries to be sent to the viewer.
-
-        References
-        ----------
-
-        .. [1] http://ds9.si.edu/ref/xpa.html
+        The `XPA access point <http://ds9.si.edu/ref/xpa.html>`_ of the ds9 image
+        viewer lets commands and queries to be sent to the viewer.
```

Comment:
```suggestion
        The XPA access point of the ds9 image viewer lets commands and
        queries to be sent to the viewer.
```

Turns out this link is now a 404. I've asked Bill if there's a valid link, but we can also just drop it here.